recipes/pynwb/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pynwb" %}
+{% set version = "0.1" %}
+{% set org = "NeurodataWithoutBorders" %}
+{% set branch = "dev" %}
+{% set sha256 = "e97fd13b714303ea1fc4bff1cec52a3d2eaa0ad33f45265651774af8409010fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url:  https://github.com/{{ org }}/{{ name }}/archive/{{ branch }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - h5py
+    - numpy
+    - scipy
+    - ruamel.yaml
+    - python-dateutil
+    - six
+    - requests
+
+test:
+  imports:
+    - pynwb
+
+about:
+  home: http://github.com/{{ org }}/{{ name }}
+  license: Custom
```

Comment:
It looks like ` BSD-3-Clause` license. 

https://opensource.org/licenses/BSD-3-Clause